app.css

```diff
@@ -0,0 +1,193 @@
+@import url('https://fonts.googleapis.com/css2?family=Work+Sans:wght@500&display=swap');
+
+body {
+    font-family: 'Work Sans';
+    font-style: normal;
+    font-weight: 600;
+    font-size: 30px;
+    line-height: 35px;
+    letter-spacing: 0.1em;
+
+    color: #000000;
+    background: #FFE4C2;
+}
+
+:root{
```

Comment:
why root and not light theme? Please let me know your thoughts here.

Reply:
:root can be useful for declaring global CSS variables and the :root CSS pseudo-class matches the root element of a tree representing the document also if I use the light theme as a class instead of :root I have to rewrite a conditions to specify colours and write a separate function to switch  Theme So I thought make use of :root set a default theme as white if any other theme matches from device then switch to it would be better here !